whatsapp_archive.py

```diff
@@ -272,31 +272,21 @@ def FormatHTML(data):
             {% for _, _, body, media in messages %}
                 {% if media is not none %}
                     {% if "IMG" in media %}
-                        <li>
```

Comment:
Is the generated HTML still valid? 

Looking at the HTML standard…
https://html.spec.whatwg.org/multipage/grouping-content.html#the-ol-element
"[Content model](https://html.spec.whatwg.org/multipage/dom.html#concept-element-content-model):
Zero or more [li](https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element) and [script-supporting](https://html.spec.whatwg.org/multipage/dom.html#script-supporting-elements-2) elements."

If you're looking to do something with the aesthetics, perhaps you can add a class tag to the &lt;li&gt; element and use CSS to give it the look you want?

Reply:
There is no problem. The HTML will work correctly because of "a", "img" and "audio" tags.

Reply:
I am testing the result with using a test file, and everyting look perfect.